Sources/Subs.php

```diff
@@ -5120,25 +5120,7 @@ function smf_chmod($file, $value = 0)
  */
 function isValidIP($IPString)
 {
-	static $existFilterVar;
-	
-	if (empty($existFilterVar))
-		$existFilterVar = function_exists('filter_var');
-	
-	if($existFilterVar === true)
-	{
-		if (filter_var($IPString,FILTER_VALIDATE_IP))
-			return true;
-		else
-			return false;
-	}
-	else
-	{
-		if (@inet_pton($IPString))
-			return true;
-		else
-			return false;
-	}
+	return filter_var($IPString,FILTER_VALIDATE_IP);
```

Comment:
Filter_var is an optional(default on) addon in php,
you are sure about to do this without backup?
And filter_var return a string and not boolean,
i'm not sure if this harm somewhere or not.
I think the min is to update the function discription.